src/sentry/celery.py

```diff
@@ -14,8 +14,8 @@
     [
         # basic tasks that must be passed models still
         "sentry.tasks.process_buffer.process_incr",
-        "sentry.tasks.process_resource_change_bound",
-        "sentry.tasks.sentry_apps.send_alert_event",
+        "sentry.sentry_apps.tasks.sentry_apps.process_resource_change_bound",
```

Comment:
Q: what is pickling/pickle tasks -> Like my google research has told me pickle is a serializer for celery tasks. So is pickling when you pass in a too complex object to get serialized? Is that bad because serializing an object is really expensive?